### PR TITLE
Release 1.114.0

### PR DESCRIPTION
**Pull request by SiobhyB**
Release for Gutenberg Mobile 1.114.0

## Related PRs

- Gutenberg: https://github.com/WordPress/gutenberg/pull/59461
- iOS: https://github.com/wordpress-mobile/WordPress-iOS/pull/22733
- Android: https://github.com/wordpress-mobile/WordPress-Android/pull/20339

## Changes

* [Introduce VideoPress v5 support, to fix issues using video block with dotcom and Jetpack sites](https://github.com/Automattic/jetpack/pull/35637)
* [Prevent crash when autoscrolling to blocks](https://github.com/WordPress/gutenberg/pull/59110)
* [Remove opacity change when images are being uploaded](https://github.com/WordPress/gutenberg/pull/59264)
* [Media & Text blocks correctly show an error message when the attached video upload fails](https://github.com/WordPress/gutenberg/pull/59288)

## Test plan

Once the installable builds of the main apps are ready, perform a quick smoke test of the editor on both iOS and Android to verify it launches without crashing. We will perform additional testing after the main apps cut their releases.

## Release Submission Checklist

- [ ] Verify Items from test plan have been completed
- [x] Check if `RELEASE-NOTES.txt` is updated with all the changes that made it to the release. Replace `Unreleased` section with the release version and create a new `Unreleased` section.
- [x] Check if `gutenberg/packages/react-native-editor/CHANGELOG.md` is updated with all the changes that made it to the release. Replace `## Unreleased` with the release version and create a new `## Unreleased`.
- [ ] Bundle package of the release is updated.

### CODE REVIEW

File: ios-xcframework/XCFrameworkScaffold.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -389,11 +385,7 @@
 					"$(inherited)",
 					_LIBCPP_ENABLE_CXX17_REMOVED_UNARY_BINARY_FUNCTION,
 				);
-				OTHER_LDFLAGS = (
-					"$(inherited)",
-					"-Wl",
-					"-ld_classic",
-				);
+				OTHER_LDFLAGS = "$(inherited)";
```
Comment (geriux) [inline ios-xcframework/XCFrameworkScaffold.xcodeproj/project.pbxproj]: Similar to what I [share here](https://github.com/WordPress/gutenberg/pull/59461#pullrequestreview-1908779207), not sure why this is changing 🤔

Comment (SiobhyB) [reply ios-xcframework/XCFrameworkScaffold.xcodeproj/project.pbxproj]: Building on what I found [here](https://github.com/WordPress/gutenberg/pull/59461#discussion_r1507593040), if I revert these changes locally and run `bundle exec pod install --repo-update`, the lines are removed again. I assume that's why they were removed by the release script and that it'd be okay to continue with the release with these changes. @fluiddot, could you verify whether that seems right?

**Review by geriux — APPROVED** [review]
LGTM!

File: ios-xcframework/XCFrameworkScaffold.xcodeproj/project.pbxproj (hunk 1 above)
Comment (SiobhyB) [reply ios-xcframework/XCFrameworkScaffold.xcodeproj/project.pbxproj]: Following the conversation [here](https://github.com/WordPress/gutenberg/pull/59461#discussion_r1507679647), these changes were being reverted as my local command line tools were not using Xcode 15. I've updated that now, and confirmed these changes are no longer happening when running `bundle exec pod install --repo-update`. I've reverted these changes in https://github.com/wordpress-mobile/gutenberg-mobile/pull/6687/commits/29272135494d34e63df1e198fa3474acd1ec9659.

**Comment by derekblank** [comment]
## 🟢 Writing Flow checklist - Android

* **Build:** jpandroid-24.4-rc-1
* **Device:** Samsung Galaxy S20 FE 5G

#### General
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/copy-paste.md#tc001) - Paste formatted text copied from website
#### Rich Text Format
- [x] [TC007](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc007) - Test format detection under the cursor
- [x] [TC009](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc009) - Test autocorrection doesn't apply formatting to Heading
- [x] [TC010](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc010) - Test autocorrection doesn't remove formatting from Heading
##### Splitting and merging
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc001) - Merge after writing
  - [x] Paragraph
  - [x] Heading
- [x] [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc002) - Merge after selection
  - [x] Paragraph
  - [x] Heading
- [x] [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc003) - Merge after deleting text
  - [x] Paragraph
  - [x] Heading
- [x] [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc004) - Merge after deleting all
  - [x] Paragraph
  - [x] Heading
- [x] [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc005) - Merge multiple blocks
  - [x] Paragraph
  - [x] Heading

**Comment by derekblank** [comment]
## 🟢    Unsupported Block Editor - Android

* **Build:** jpandroid-24.4-rc-1
* **Device:** Samsung Galaxy S20 FE 5G

- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc001) - User can edit unsupported blocks on Simple WP.com sites
- [x] [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc002) - User can discard edits to an unsupported block on Simple WP.com sites
- [x] [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc003) - Editing unsupported blocks is allowed on Gutenberg-enabled Atomic sites
- [x] [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc004) - Editing unsupported blocks is disallowed on Classic-enabled Atomic sites
- [x] [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc005) - Editing unsupported blocks is enabled on self-hosted sites accessed via Jetpack
- [x] [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc006) - Editing unsupported blocks is disallowed on self-hosted sites access via their own username and password

**Comment by derekblank** [comment]
## 🟢   Functionality Test Suite 4 (Android)

* **Build:** jpandroid-24.4-rc-1
* **Device:** Samsung Galaxy S20 FE 5G

### Group - 2

- [x] Group - Check if nested Placeholder block can be replaced - [TC010](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/group.md#tc010)

### Buttons - 4

- [x] Buttons block - The newly created Buttons has set background color to the theme-default color in preview - [TC017](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/buttons.md#tc017)

### Buttons - 5

- [x] Buttons block - Render theme colors - [TC015](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/buttons.md#tc015)
- [x] Buttons block - Settings: Change Button border radius - [TC011](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/buttons.md#tc011)

### Buttons - 7

- [x] Buttons block - Toolbar link button is active when Button has link - [TC018](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/buttons.md#tc018)

### Editor Theme - 2
- [x] Custom Gradients - Check that custom gradients load in the editor - [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/editor-theme.md#tc004)
- [x] Offline Support - [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/editor-theme.md#tc005)

### Color Settings - 2
- [x] Customize gradient color - [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/color-settings.md#tc005)
- [x] Custom color picker - [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/color-settings.md#tc006)

### VideoPress - 1
- [x] VideoPress block - Upload a video - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/jetpack/videopress.md#tc001)
- [x] VideoPress block - Close post with an ongoing video upload - [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/jetpack/videopress.md#tc004)

**Comment by derekblank** [comment]
## 🟢   Incoming Changes (Android)

* **Build:** jpandroid-24.4-rc-1
* **Device:** Samsung Galaxy S20 FE 5G

- [x] [Introduce VideoPress v5 support, to fix issues using video block with dotcom and Jetpack sites](https://github.com/Automattic/jetpack/pull/35637)
- [x]  [Prevent crash when autoscrolling to blocks](https://github.com/WordPress/gutenberg/pull/59110)
- [x]  [Remove opacity change when images are being uploaded](https://github.com/WordPress/gutenberg/pull/59264)
- [x]  [Media & Text blocks correctly show an error message when the attached video upload fails](https://github.com/WordPress/gutenberg/pull/59288)

**Comment by derekblank** [comment]
## 🟢 Writing Flow checklist - iOS

* **Build:** [24.4.0.0](https://github.com/wordpress-mobile/WordPress-iOS/releases/tag/24.4.0.0)
* **Device:** iPhone SE

#### General
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/copy-paste.md#tc001) - Paste formatted text copied from website
#### Rich Text Format
- [x] [TC007](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc007) - Test format detection under the cursor
- [x] [TC009](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc009) - Test autocorrection doesn't apply formatting to Heading
- [x] [TC010](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc010) - Test autocorrection doesn't remove formatting from Heading
##### Splitting and merging
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc001) - Merge after writing
  - [x] Paragraph
  - [x] Heading
- [x] [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc002) - Merge after selection
  - [x] Paragraph
  - [x] Heading
- [x] [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc003) - Merge after deleting text
  - [x] Paragraph
  - [x] Heading
- [x] [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc004) - Merge after deleting all
  - [x] Paragraph
  - [x] Heading
- [x] [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc005) - Merge multiple blocks
  - [x] Paragraph
  - [x] Heading

**Comment by derekblank** [comment]
## 🟢    Unsupported Block Editor - iOS

* **Build:** [24.4.0.0](https://github.com/wordpress-mobile/WordPress-iOS/releases/tag/24.4.0.0)
* **Device:** iPhone SE

- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc001) - User can edit unsupported blocks on Simple WP.com sites
- [x] [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc002) - User can discard edits to an unsupported block on Simple WP.com sites
- [x] [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc003) - Editing unsupported blocks is allowed on Gutenberg-enabled Atomic sites
- [x] [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc004) - Editing unsupported blocks is disallowed on Classic-enabled Atomic sites
- [x] [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc005) - Editing unsupported blocks is enabled on self-hosted sites accessed via Jetpack
- [x] [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc006) - Editing unsupported blocks is disallowed on self-hosted sites access via their own username and password

**Comment by derekblank** [comment]
## 🟢   Functionality Test Suite 5 (iOS)

* **Build:** [24.4.0.0](https://github.com/wordpress-mobile/WordPress-iOS/releases/tag/24.4.0.0)
* **Device:** iPhone SE

Video block - 1
- [x] Video block - Insert video from device (failing) - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/video.md#tc001)
- [x] Video block - Close/Re-open post with an ongoing video upload - [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/video.md#tc004)

Video block - 2
- [x] Video block - Insert video from device (cancel) - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/video.md#tc002)
- [x] Video block - Close post with an ongoing video upload - [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/video.md#tc005)
- [x] Video block - Add Caption - [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/video.md#tc003)

File block - 1
- [x] File block - Upload completes successfully from the device library - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc001)
- [x] File block - Insert completes successfully from the WordPress Media library - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc002)
- [x] File block - Upload fails from the device library - [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc003)
- [x] File block - Stop the file upload before it completes - [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc004)

File block - 2
- [x] File block - Close the post with an ongoing file upload - [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc005)
- [x] File block - Replace the selected files - [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc006)
- [x] File block - Change the alignment in the File Block - [TC007](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc007)
- [x] File block - The Link To Block Setting should open - [TC008](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc008)

File block - 3
- [x] File block - Open link in new tab - [TC009](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc009)
- [x] File block - Show/hide download button (Block Settings only) - [TC010](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc010)
- [x] File block - Copy file URL - [TC011](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc011)
- [x] File block - Close/Re-open post with an ongoing file upload - [TC012](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc012)

Audio block - 1
- [x] Audio block - Upload completes successfully from the device library - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/audio.md#tc001)
- [x] Audio block - Insert completes successfully from the WordPress Media library - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/audio.md#tc002)

Audio block - 2
- [x] Audio block - Upload fails from the device library - [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/audio.md#tc003)
- [x] Audio block - Stop the audio file upload before it completes - [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/audio.md#tc004)

Audio block - 3
- [x] Audio block - Close the post with an ongoing audio file upload - [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/audio.md#tc005)
- [x] Audio block - Replace the selected files - [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/audio.md#tc006)
- [x] Audio block - Change the alignment in the Audio Block - [TC007](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/audio.md#tc007)

Audio block - 4
- [x] Audio block - Close/Re-open post with an ongoing audio file upload - [TC010](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/audio.md#tc010)

Synced patterns - 1
- [x] Synced patterns - Add pattern - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/synced-patterns.md#tc001)
- [x] Synced patterns - Detach pattern - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/synced-patterns.md#tc002)

Synced patterns - 2
- [x] Synced patterns - Deleted pattern - [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/synced-patterns.md#tc003)
- [x] Synced patterns - Recursive block rendering - [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/synced-patterns.md#tc004)

Embed block - 1
- [x] Embed block - Inline previews are rendered correctly for the top 5 most-used providers - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/embed.md#tc001)

Embed block - 2
- [x] Embed block - Inline previews are rendered accordingly for each alignment option - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/embed.md#tc002)

**Comment by derekblank** [comment]
## 🟢   Incoming Changes (iOS)

* **Build:** [24.4.0.0](https://github.com/wordpress-mobile/WordPress-iOS/releases/tag/24.4.0.0)
* **Device:** iPhone SE

- [x] [Introduce VideoPress v5 support, to fix issues using video block with dotcom and Jetpack sites](https://github.com/Automattic/jetpack/pull/35637)
- [x]  [Prevent crash when autoscrolling to blocks](https://github.com/WordPress/gutenberg/pull/59110)
- [x]  [Remove opacity change when images are being uploaded](https://github.com/WordPress/gutenberg/pull/59264)
- [x]  [Media & Text blocks correctly show an error message when the attached video upload fails](https://github.com/WordPress/gutenberg/pull/59288)

**Pull request merged**